src/network/networkd-fdb.c

```diff
@@ -146,6 +148,18 @@ static int fdb_entry_configure(Link *link, FdbEntry *fdb_entry) {
                         return log_link_error_errno(link, r, "Could not append NDA_VLAN attribute: %m");
         }
 
+        if (fdb_entry->ifname) {
+                Link *l;
+
+                r = link_get_by_name(link->manager, fdb_entry->ifname, &l);
+                if (r < 0)
+                        return log_link_error_errno(link, r, "Failed to resolve ifname %s: %m", fdb_entry->ifname);
+
+                r = sd_netlink_message_append_u32(req, NDA_IFINDEX, l->ifindex);
+                if (r < 0)
+                        return log_link_error_errno(link, r, "Could not append NDA_IFINDEX attribute: %m");
+        }
```

Comment:
I think we should skip to configure the entry rather than return failure when the device does not exist yet.
And, configure the entry when the corresponding interface comes.

Such a mechanism should also be used for `MultipathRoute=`, and not sure but maybe `OutgoingInterface=` and `IncomingInterface=` in `[RoutingPolicyRule]`.

Reply:
https://github.com/systemd/systemd/pull/17248#discussion_r510582939. Then why did you made this comment